taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/3_sankIrNakANDam/02_vinAyakakalpaH.md

```diff
@@ -2,50 +2,116 @@
+अथ प्रयोगः ॥
 
-629
+> ऋद्धिसिध्यादिकामो मासिमास शुक्लपक्षस्य चतुर्थ्यां पञ्चम्यां वा विवाहाद्यभ्युदयकर्मसु दाहणे सङ्कटे प्राप्ते तन्निवृत्तिकामश्च विनायकशान्तिं कुर्यात् ॥ कर्ता पूर्वेद्युः कृतैकभुक्तोऽपरेद्युर्नित्यकर्म कृत्वा देशकालौ सङ्कीर्त्य समस्त मनोरथसिध्यर्थं (अमुककर्मणि निर्विघ्नतासिध्यर्थं) श्रीविनायकप्रीत्यर्थं विनायकशान्तिं करिष्ये,
 
-बो० ०
+इति सङ्कल्प्य ।
 
-SSANS-
+> तदङ्गतया | गणेशपूजन-नान्दीसमाराधन-पुण्याहवाचनानि कुर्यात् । पुण्याहवाचने ‘विनायकः प्रीयताम्'
+
+इति विशेषः ॥
+
+> देवयजनोल्लेखनप्रभृत्यग्निं प्रतिष्ठाप्य, विनायकशान्तिहोमः कर्म, वरदो नामाऽग्निः, सविता देवता, चरुर्हविः, इत्याद्यन्वाधाय प्रोक्षणीसंस्कारादि सवित्रे सिद्धमोदनं निरुप्य प्रोक्ष्य प्रतिष्ठिताभिधारणान्तं कुर्यात् ॥ अग्रेणाऽग्निमष्टदलोपरि स्वस्तिके कलशं प्रतिष्ठाप्य, तस्मिन् (बिल्वसार) सुवर्णादिनिर्मितां दक्षिणावर्तशुण्डलयुतां विनायकप्रतिमां, पूर्णपात्रे--सूत्रपञ्चकनिर्मितं कङ्कणञ्च विन्यस्योपोत्थाय "विघ्न विघ्नेश्वरागच्छ विध्नेत्येव नमस्कृत । अविघ्नाय भवान्सम्यक्सदाऽस्माकं भव प्रभो ॥
+ॐ भूर्भुवः सुवः विनायकमावाहयामि” इत्यावाह्य ॥' इमा आपः शिवाः शिवतमाः शान्ताः शान्ततमाः पूताः पूततमाः पुण्याः पुण्यतमा मेध्या मेध्यतमा जुष्टा जुष्टतमा अमृता अमृतरसाः पाद्याः प्रतिगृह्यंतां प्रतिगृह्णातु भगवान्विनायको विनायकाय नमः'
+
+इति पाद्यं ॥
+
+> 'इमा आप:० अमृतरसा अर्ध्याः प्रतिगृह्यतां० नमः'
+
+इत्यर्घ्यं० ।।
+
+> एवमेव 'अर्हणीयाः०, अभिषेचनीयाः०, आचमनीयाः०, मार्जनीयाः०' दत्वाऽथ 'विनायकाय नमः' इति मन्त्रेण गन्ध-पुष्प-धूप-दीपांश्च तूष्णीं वा दद्यात् ॥
+
+> अथ विनायकमुपतिष्ठते-" भूपतये नमो भुवनपतये नमो भूतानां पतये नमः”
```

Comment:
"अथ विनायकमुपतिष्ठते-" इत्य् उद्धारो (quotation) नास्ति। अतः, 

```
अथ विनायकमुपतिष्ठते-

> भूपतये नमो भुवनपतये नमो भूतानां पतये नमः
```

एवं सर्वासु सञ्चिकासु परीक्ष्य समीकरोतु।

Reply:
भूपतये नमो, भुवनपतये नमो भूतानां पतये नम:
अत्र आद्ये तथा अवसाने उद्धारौ स्तः । 
पङ्क्ते: आदे न करणीयम् इति मन्ये ॥
सर्वासु पञ्चिकासु परीक्षितम्

Reply:
भवता नावगतो दोषः। "अथ विनायकमुपतिष्ठते-" is not part of the quotation. It should not be put within the quotation line, but before it.